my-app/src/app/courses-page/course-item/course-item.component.css

```diff
@@ -187,4 +187,115 @@
     .delete-button {
         margin-left: 0;
     }
-}
\ No newline at end of file
+}
+
+::ng-deep .cdk-overlay-connected-position-bounding-box{
+    position: absolute;
+    background-color: rgb(203, 208, 212);
+}
+
+::ng-deep .cdk-overlay-pane {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    bottom: unset !important;
+    right: unset !important;
+    margin-right: -50%;
+    transform: translate(-50%, -50%);
+}
+
+.delete-popup {
+    height: 240px;
+    width: 400px;
+    border-radius: 5px;
+    background-color: rgb(163, 164, 165);
+    box-shadow: -10px 10px 10px 0 #333;
+}
+
+.delete-popup-header {
+    line-height: 24px;
+    letter-spacing: 0.1px;
+    color: rgb(46, 49, 49);
+    font-size: 24px;
+    font-family: Arial, Helvetica, sans-serif;
+    text-align: center;
+    font-weight: 600;
+    height: 60px;
+    padding: 18px 24px;
+    display: flex;
+    justify-content: space-between;
```

Comment:
you have a different indentation for styles. 2 and 4 tabs. Please use formatted so the code is well-formatted with correct indentations. I would recommend 2 space identation=)